package.json

```diff
@@ -25,7 +25,9 @@
   "scripts": {
     "clean": "rm -rf build",
     "build": "npm run clean; tsc",
-    "test": "TZ=utc mocha"
+    "test": "TZ=utc mocha",
+    "device": "npm run build; node build/Device.js",
+    "list-c-nodes": "dns-sd -B _matterc._udp"
```

Comment:
This more fits into a troubleshooting section in readme because this toll pot needs to be installed before it can be used.